src/styles/globalStyle.js

```diff
@@ -8,15 +8,6 @@ const GlobalStyle = createGlobalStyle`
     font-family: ${(props) => props.theme.fonts[1]};
     color: ${(props) => props.theme.colors.gray.main};
   }
-  h1, h2, h3, h4, h5, h6 {
```

Comment:
I think it makes sense to define global styles for the headline. Afaik they are shared throughout the app. Was there something particular that made you delete them?